qt/place_panel.cpp

```diff
@@ -12,15 +11,15 @@
 #include <QtWidgets/QVBoxLayout>
 #include <QtWidgets/QLabel>
 #include <QtWidgets/QPushButton>
-#include <QtWidgets/QDialog>
 
 #include <sstream>
 #include <string>
 
 namespace
 {
-static int constexpr kMaxLengthOfPlacePageDescription = 500;
-static int constexpr kMinWidthOfShortDescription = 390;
+const int kMaximumPanelWidth = 390;
```

Comment:
constexpr here and below, it is not clear what's the dimension of the variable (pixels?)